doc/whats_new/v0.22.rst

```diff
@@ -484,6 +484,47 @@ Changelog
   :func:`~utils.estimator_checks.parametrize_with_checks`, to parametrize
   estimator checks for a list of estimators. :pr:`14381` by `Thomas Fan`_.
 
+- |API| `requires_positive_X` estimator tag (for models that require
+  X to be non-negative) is now used by `check_estimator` to make sure
+  a proper error message is raised if X contains some negative entries.
+  :pr:`14680` by :user:`Alex Gramfort <agramfort>`.
+
+- |Enhancement| :func:`utils.safe_sparse_dot` works between 3D+ ndarray and
```

Comment:
This entry seems to be a side-effect from resolving some merge-conflict, isn't it?